### DISCUSSION TITLE
建议优化一下搜索栏的样式

### DISCUSSION THREAD

**Discussion by Lynricsy** [the1812/Bilibili-Evolved#1522] [功能建议 / Ideas]
可以参考一下<a href="https://github.com/hakadao/bilibili-simple-home">这个</a>，感觉这个做的很漂亮
目前这两个同时使用会导致浏览器崩溃

**Comment by Lynricsy** [comment]
现在的主页做的已经很好了，但是顶部搜索栏的样式还可以参考一下这个

**Comment by Geeyun-JY3** [comment]
之前兼容过：#270、#439、#514，我这边也一直在用这个样式，没什么问题。提供一下更多信息？如浏览器版本、脚本版本什么的

**Comment by Lynricsy** [comment]
浏览器:
Microsoft Edge
版本 90.0.782.0 (官方内部版本) canary (64 位)
脚本是最新的

**Comment by Lynricsy** [comment]
不过我的主要的意思不是说想兼容

**Comment by Lynricsy** [comment]
是说能不能优化一下这个脚本的主页

**Comment by Lynricsy** [comment]
因为脚本的这个主页除了搜索框之外都很好看

**Reply by the1812** [reply]
但是我那是主页 跟搜索框没啥关系 (或者说根本就没有特制的搜索框
现在的搜索框在顶栏上, 所有主站页面都有

**Reply by Coulomb-G** [reply]
自定义顶栏的搜索框也觉得不好看吗?

**Reply by Lynricsy** [reply]
> 但是我那是主页 跟搜索框没啥关系 (或者说根本就没有特制的搜索框
> 现在的搜索框在顶栏上, 所有主站页面都有

不过顶栏的搜索栏也可以改吧
就是用一个半透明（最好毛玻璃）的圆角矩形就挺好看的

**Reply by Lynricsy** [reply]
现在不就已经改了吗（指跟原版不一样）

**Reply by the1812** [reply]
顶栏后面是一整条的背景, 搜索栏半透明透不到后面去的

**Reply by Lynricsy** [reply]
> 顶栏后面是一整条的背景, 搜索栏半透明透不到后面去的

那也没关系啊，白色的也比现在好看（我感觉）

**Reply by the1812** [reply]
那你就用 Stylus 改一下吧, 给搜索框去掉边框加个背景色

**Reply by Lynricsy** [reply]
> 那你就用 Stylus 改一下吧, 给搜索框去掉边框加个背景色

我要是自己会改的话，也不用发这个了吧（是我太菜了）我根本不会写stylus的css啊

**Reply by Coulomb-G** [reply]
你是不是开着主题色填充?换成透明填充看看?
还不行的话直接用PS画一下你想要的样子,我们这边可以告诉你css
